modules/router/templates/router_include.conf.erb

```diff
@@ -63,7 +63,9 @@ location ~ ^/contact/govuk/(|service-feedback|problem_reports|foi|page_improveme
 location / {
   <%= scope.function_template(['router/fragments/_basic_auth.erb']) -%>
 
+  # Enable asset domain preconnect and server push for HTTP/2-capable clients
   add_header Link "<https://assets.<%= @app_domain %>>; rel=preconnect; crossorigin";
+  http2_push_preload on;
```

Comment:
The nginx docs say:
> Enables automatic conversion of preload links specified in the “Link” response header fields into push requests.
So does the line above need `rel=preconnect` changing to `rel=preload`?

Reply:
They're actually separate things - the first line hints to the browser to set up the connection to assets even before it has worked out it needs to request anything from there. The second line proactively pushes assets to the browser that have been marked by individual apps.

Reply:
Maybe add a comment, or even just a blank line, to make it clearer that these lines deal with different things.

Reply:
Separated with comments